### ISSUE TITLE
Audio Crackling on bluetooth headphones

### ISSUE THREAD

**Issue by machurash** [zen-browser/desktop#3778]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

ONLY ON YOUTUBE PLAYER

Whenever you unpause or skip the video using your arrow keys or by pressing on the progress bar with your mouse to watch your video on a new timestamp you always get cracking noises for ~1 second and rarely the audio gets bugged and until you pause the video for 1 second the audio is like its in 8 bits from all the cracking sounds. This happens only on youtube and only on Zen Browser. With a pair of wired headsets I had no trouble at all.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.1

### What platform are you seeing the problem on?

Windows

**Comment by mort666** [comment]
Well I've given up using zen with bluetooth Audio on ArchLinux, the audio is either noisy or for some reason Zen just cannot handle pipewire routed audio?? I've literally zero clue why when a bluetooth audio interface is linked up with Zen the audio is about 75-80% of the time never going to get routed to the bluetooth device and most of the time it gets sent to a random HDMI/DisplayPort hardware interface on the GPU. When it does get routed is super noisy..

I've had this happening now since 1.0.1-a.22

**Comment by djorium** [comment]
And have you tried with non-bluetooth peripherals?

**Comment by mort666** [comment]
> And have you tried with non-bluetooth peripherals?

Yes, basically anything none wireless is okay for me. So audio via DisplayPort/HDMI or direct wired to speakers from the motherboards onboard audio is okay it seems it is just audio that ultimately would go to a wireless connection, I was just trying this evening to have Airplay to a HomePod via WiFi and that also seems to be an issue.. Should also note this is on two different machines, both running ArchLinux and HyprLand.. One a Laptop the other is a Desktop, both AMD Ryzen systems with Zen 3 on the laptop and Zen 5 on the Desktop and both Asus hardware ROG G15 Laptop and ProArt X870E Creator for the desktop. There is any issue btw, with other browsers...

**Comment by machurash** [comment]
Hello, after the 1.7.2b update the issue is still here, when changing
videos on youtube you hear a small crack. You hear that sound when the
video resumes after pausing, when you skip the video by clicking on the
progress bar (for some reason this is the worst cause the cracklings there
are like a bag of popcorn in a microwave while the others are just like
pressing a button) and by skipping the video with the arrow keys

**Comment by asidius** [comment]
I have the same issue, but it is with 2.4ghz wireless headset. 

The issue also happens in all variations of firefox browsers and only in firefox based browsers. 

It's definitely a firefox issue so expect a fix in 48 business years from mozilla.

**Comment by glockjt** [comment]
I have this same issue just with audio out of my Mac. I tested the same audio from a Google Meet with other browsers and the other browsers sounded great but in Zen, it was crackling and sounded like it was coming from a tunnel.

**Comment by AndriesAndreiIonut** [comment]
Hello, I just updated to 1.9b, the problem is still there unfortunately. It may be my mind playing tricks on me since i haven't used the wireless headphones on this browser since 1.7.2b but now it seems milder than before but it definitely is still there

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=Audio%20Crackling%20on%20bluetooth%20headphones%0A
